SofaImGui/src/SofaImGui/windows/Profiler.cpp

```diff
@@ -23,119 +23,96 @@
 #include <imgui_internal.h> //imgui_internal.h is included in order to use the DockspaceBuilder API (which is still in development)
```

Comment:
you changed the format of the file. It's difficult to review as the true differences are mixed with format changes. Could you revert the format changes only?